README.md

```diff
@@ -39,31 +54,75 @@ I like emoji. I like ’em a lot. Programming, code, geeks/nerds, open source, a
 
 Only use the following Git Commit Messages. Simple and small foot print is the key here.
 
-1. `📦 NEW: IMPERATIVE_MESSAGE_GOES_HERE`
+---
```

Comment:
This entire format is bulky and confusing. Compare both views side by side. I am all for less is more. But not interested in adding more complexity to the instructions.